src/database/createAccount.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import firebase from 'firebase/app';
+import FirebaseApp from './config';
+import firebaseApp from './config';
+
+const createAccount = (props) => {
+    const email = props.email;
+    const password = props.password;
+    const userName = props.userName;
+
+    const informationDataAccount = {
+        uid: email,
+        userName: userName,
+    };
+
+    const errorCreateAccount = () => {
+        
+    };
+
+    const successCreateAccount = () => {
+
+    };
```

Comment:
What do those functions do?

Reply:
The first function `errorCreateAccount`: When the user attempt to create a new account and the account already exists or the password is weak then the function sends an error message.

The second function `successCreateAccount`: When the user attempt to create a new account and already successfully create an account the function sends a successful message.